src/components/FormHelpMessage.tsx

```diff
@@ -23,11 +25,29 @@ type FormHelpMessageProps = {
 
     /** Whether to show dot indicator */
     shouldShowRedDotIndicator?: boolean;
+
+    /** Whether should render error text as HTML or as Text */
+    shouldParseMessage?: boolean;
```

Comment:
Could you please rename this to `shouldRenderErrorAsHTML` so it is more explicit?

Reply:
Renamed to `shouldRenderMessageAsHTML` as this correlates better with other `message` prop